src/game.rs

```diff
@@ -390,8 +390,7 @@ impl Game {
                 for _ in 0..=num_rotations {
                     m = Tile::rotate_matrix_clockwise(&m);
                 }
-
-                let rotated_tile = &Tile::from_matrix(m, t.name.clone(), new_pos);
+                let rotated_tile = &Tile::from_matrix(m, t.name.clone(), new_pos, num_rotations);
                 self.game_state.tiles.push(rotated_tile.clone());
```

Comment:
You wouldn't need to clone rotated_tile if you didn't make rotated_tile a reference in the first place. As in, get rid of the `&` on line 393 and then you can get rid of the `clone()` on line 394.

Reply:
Sounds about right. Something something borrowed later on 398 so clone is still necessary but the ampersand isn't